content/storage/postgres/docker-compose-for-postgres.md

```diff
@@ -38,9 +38,16 @@ services:
 
 # Create name volumes managed by Docker to not lose data when remove a container
 # https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes
+
+# You can also specify a volume driver such as pxd (Portworx) to achieve high performance read/write for container storage volumes.
```

Comment:
Curiosity, if it high performance why don't this very is set by default?

Reply:
because it is specific case if only for developer zone, you don't be required for it